Build/libHttpClient.Linux.C/libHttpClient.Linux.C.vcxproj

```diff
@@ -0,0 +1,187 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
is this generated from a template?

Reply:
It was generated from creating empty projects in visual studio.